libclingo/clingo.h

```diff
@@ -1989,22 +1989,26 @@ enum clingo_statistics_type {
 //! Corresponding type to ::clingo_statistics_type.
 typedef int clingo_statistics_type_t;
 
-//! Handle for to the solver statistics.
+//! Handle for the solver statistics.
 typedef struct clingo_statistic clingo_statistics_t;
 
+//! Callback to add user defined statistics.
+typedef bool (*clingo_set_user_statistics) (clingo_statistics_t* stats, void* data);
```

Comment:
All typedefs in clingo.h have suffix `_t`,  callbacks have suffix `_callback_t`. I would recommend to use `clingo_statistics_update_callback_t`. Other recommendations for names are welcome.